main.go

```diff
@@ -91,6 +93,17 @@ func main() {
 		Namespace:              watchNamespace,
 	}
 
+	// Add support for MultiNamespace set in WATCH_NAMESPACE (e.g ns1,ns2)
+	if strings.Contains(watchNamespace, ",") {
```

Comment:
If `watchNamespace` specifies multiple namespaces then we need to change the `cluster.New` calls to look something like:

```go
c, err := cluster.New(cfg, func(o *cluster.Options) {
	o.Scheme = scheme
	o.Namespace = ""
	o.NewCache = cache.MultiNamespacedCacheBuilder(strings.Split(watchNamespace, ","))
})
```
We also need to change the query that fetches ClientConfigs. It needs to specify multiple/separate namespace options.

After those changes, we need to deploy the operator with a ClusterRole and ClusterRoleBinding instead of the Role and RoleBinding.

Lastly, we need updated test coverage.